include/game/multiplayer/indicators.h

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
Remember this needs the guards and needs to include `global.h` to get the `u8` type